src/ifcparse/Argument.h

```diff
@@ -20,59 +20,59 @@
 #ifndef ARGUMENT_H
 #define ARGUMENT_H
 
-#include <set>
-#include <string>
-#include <vector>
-#include <sstream>
-#include <algorithm>
-
-#include "ifc_parse_api.h"
 #include "ArgumentType.h"
+#include "ifc_parse_api.h"
 
-#include <boost/shared_ptr.hpp>
+#include <algorithm>
 #include <boost/dynamic_bitset.hpp>
 #include <boost/logic/tribool.hpp>
+#include <boost/shared_ptr.hpp>
+#include <set>
+#include <sstream>
+#include <string>
+#include <vector>
 
 class aggregate_of_instance;
 class aggregate_of_aggregate_of_instance;
 
 namespace IfcUtil {
-	class IfcBaseClass;
+class IfcBaseClass;
 
-    IFC_PARSE_API const char* ArgumentTypeToString(ArgumentType argument_type);
+IFC_PARSE_API const char* ArgumentTypeToString(ArgumentType argument_type);
 
-	/// Returns false when the string `s` contains character outside of {'0', '1'}
-	IFC_PARSE_API bool valid_binary_string(const std::string& s);
-}
+/// Returns false when the string `s` contains character outside of {'0', '1'}
+IFC_PARSE_API bool valid_binary_string(const std::string& s);
+} // namespace IfcUtil
 
 class IFC_PARSE_API Argument {
-public:
-	virtual operator int() const;
-	virtual operator bool() const;
-	virtual operator boost::logic::tribool() const;
-	virtual operator double() const;
-	virtual operator std::string() const;
-	virtual operator boost::dynamic_bitset<>() const;
-	virtual operator IfcUtil::IfcBaseClass*() const;
+  public:
+    virtual operator int() const;
+    virtual operator bool() const;
+    virtual operator boost::logic::tribool() const;
+    virtual operator double() const;
+    virtual operator std::string() const;
+    virtual operator boost::dynamic_bitset<>() const;
+    virtual operator IfcUtil::IfcBaseClass*() const;
+
+    virtual operator std::vector<int>() const;
+    virtual operator std::vector<double>() const;
+    virtual operator std::vector<std::string>() const;
+    virtual operator std::vector<boost::dynamic_bitset<>>() const;
+    virtual operator boost::shared_ptr<aggregate_of_instance>() const;
 
-	virtual operator std::vector<int>() const;
-	virtual operator std::vector<double>() const;
-	virtual operator std::vector<std::string>() const;
-	virtual operator std::vector<boost::dynamic_bitset<> >() const;
-	virtual operator boost::shared_ptr<aggregate_of_instance>() const;
+    virtual operator std::vector<std::vector<int>>() const;
+    virtual operator std::vector<std::vector<double>>() const;
+    virtual
+    operator boost::shared_ptr<aggregate_of_aggregate_of_instance>() const;
```

Comment:
line break prop. due to statement line length

Reply:
`virtual operator boost::shared_ptr<aggregate_of_aggregate_of_instance>() const;` is +- 80 chars it seems. Can we increase that?

Reply:
LLVM and Google style use a `ColumnLimit` of 80. We could set it to 120, but there will always be a line which is just 1-2 characters too long. Or we could NOT introduce a limit for the beginning (setting it to 0) and bet on some form of self-control?

Reply:
I would choose either 120 or 0. You decide.